src/LMTR_alg.jl

```diff
@@ -167,7 +167,8 @@ function LMTR(
     end
 
     subsolver_options.ϵa = k == 1 ? 1.0e-5 : max(ϵ, min(1.0e-1, ξ1 / 10))
-    set_radius!(ψ, min(β * χ(s), Δk))
+    Δk2 = min(β * χ(s), Δk)
```

Comment:
Ok. Could we find a better name than `∆k2` though? Maybe `∆_effective` or something like that (although it's longer)?